steer_drive_controller/README.md

```diff
@@ -0,0 +1,80 @@
+## Steer Drive Controller ##
+
+Controller for a steer drive mobile base.
+
+## 仕様
```

Comment:
Please use English, I'd love to learn Japanese but I feel this is not the right place or time for it :)

Reply:
Oops, I'm so sorry. I forgot to remove Japanese stuff from here. I added a link to the English ROS Wiki page currently :)